frontend/public/components/RBAC/bindings.jsx

```diff
@@ -730,7 +730,7 @@ export const CreateRoleBinding = ({ match: { params }, location }) => {
       setActiveNamespace={setActiveNamespace}
       fixed={fixed}
       isCreate={true}
-      titleVerb={t('public~Create')}
+      titleVerbAndKind={t('public~Create RoleBinding')}
```

Comment:
Can you confirm that these don't create either a RoleBinding or ClusterRoleBinding? Is it always RoleBinding? Otherwise, we've introduced a regression where the kind labels don't match.

Reply:
> Can you confirm that these don't create either a RoleBinding or ClusterRoleBinding? Is it always RoleBinding? Otherwise, we've introduced a regression where the kind labels don't match.

@spadgett I am not sure that I understand the first sentence in your comment. No it create a binding based on the option selected. See attached screenshots for the RB and CRB created from the changes in this PR.
![Screenshot 2023-03-22 at 9 00 37 AM](https://user-images.githubusercontent.com/15249132/226918727-9925d5ab-3a45-44af-8379-32de6f13ba84.png)